topdown/http.go

```diff
@@ -613,16 +613,18 @@ func checkHTTPSendInterQueryCache(bctx BuiltinContext, key ast.Object, req *http
 		}
 	}
 
-	err = insertIntoHTTPSendInterQueryCache(bctx, key, newValue, result, size)
+	force := cacheParams != nil && cacheParams.forceCacheDurationSeconds > 0
```

Comment:
Since we are trying to override the `no-store` directive here, couldn't we just use the value of `force_cache` param to determine this. For ex, if `force_cache` is `true` override no-store, if not then decide logic based on the `no-store` directive from server.

Reply:
Indeed - is this not what this is doing? 😅 In the following function call we check:

`(!force && !canStore(resp.Header))`

..so the `no-store` directive in the response will be checked unless `force_cache` is `true`.

Reply:
I'm thinking what if we used `cacheParams != nil` itself as the override for `no-store`. We will have other cache parameters in the future apart from `forceCacheDurationSeconds`, so `cacheParams != nil` which implies `force_cache` is `true`. wdyt ?

Reply:
Oh right, so like now but skipping the extra `cacheParams.forceCacheDurationSeconds > 0` check? Yeah that I added mostly due to not knowing what the intent of `force_cache = true` together with `force_cache_duration = 0` would be. I'll use only `cacheParams != nil` then.

Reply:
Fixed 👍